src/containers/backdrop-library.jsx

```diff
@@ -27,9 +27,9 @@ class BackdropLibrary extends React.Component {
     render () {
         return (
             <LibraryComponent
+                visible
```

Comment:
I think we should go one step further and just remove the prop altogether. The right way to do it is conditionally rendering the component.